index.html

```diff
@@ -49,18 +49,19 @@ <h2>SF2</h2>
 
         Constituents:
         <ul>
-            <li><a href="https://x.com/sulaimanghori">Sulaiman Ghori</a></li>
-            <li><a href="https://x.com/sincethestudy">Brian Machado</a></li>
-            <li><a href="https://x.com/ivan_yevenko">Ivan Yevenko</a></li>
-            <li><a href="https://x.com/clayhaight">Clayton Haight</a></li>
             <li><a href="https://x.com/adamcohenhillel">Adam Cohen Hillel</a></li>
-            <li><a href="https://x.com/PabloDerre">Pablo Derre</a></li>
-            <li><a href="https://x.com/marvy_101">Dada</a></li>
             <li><a href="https://x.com/anshchopra_">Ansh Chopra</a></li>
+            <li><a href="https://x.com/marvy_101">Dada</a></li>
```

Comment:
Dada should come after *A*LEX

Reply:
Was the wrong fucking branch I created the PR from hahaha – this is the GPT´s attempt on doing ABC Ordering xd